backend/amd64/proc.ml

```diff
@@ -133,18 +144,28 @@ let hard_float_reg =
   for i = 0 to 15 do v.(i) <- Reg.at_location Float (Reg (100 + i)) done;
   v
 
+let hard_vec128_reg =
+  let v = Array.make 16 Reg.dummy in
+  for i = 0 to 15 do v.(i) <- Reg.at_location Vec128 (Reg (100 + i)) done;
```

Comment:
Are we sure they should not have the same stamps
as the registers in `hard_float_reg`?

In concrete terms, do we think
`Reg.same hard_float_reg.(i) hard_vec128_reg.(i)`
should return `true` or `false`?

Reply:
(After some discussion, we probably won't do this)